src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,67 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void requirement1() {
+        // when
+        int size = numbers.size();
+
+        // then
+        assertThat(size).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void requirement2(int number) {
+        // when
+        boolean hasValue = numbers.contains(number);
+
+        // then
+        assertThat(hasValue).isTrue();
```

Comment:
특정 값을 가지고 있는지 검증하는 메서드를 사용해보면 어떨까요?
```suggestion
        assertThat(numbers).contains(input);
```